info.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Metadata defining the behavior and requirements for this framework
+
+# expected fields in the configuration file for this engine
+configuration:
+
+# the Shotgun fields that this engine needs in order to operate correctly
+requires_shotgun_fields:
+
+# More verbose description of this item
+display_name: "Desktop Client Framework"
+description: "Provides the client used to send requests to a Desktop Server"
+
+# Required minimum versions for this item to run
+requires_shotgun_version:
+requires_core_version: "v0.18.97"
```

Comment:
Any reason you picked this version of tk-core? Is it from tk-config-basic?

Reply:
Maybe it's worth putting the latest core in there? @robblau ?

Reply:
I think I just copied another repo and stripped down everything. either the starter app or another framework.